src/drive/targets/manifest.webapp

```diff
@@ -175,5 +175,12 @@
       "verbs": ["POST"],
       "description": "Remote-doctype required to send anonymized measures to the DACC shared among mycozy.eu's Cozy."
     }
+  },
+  "accept_from_flagship": true,
+  "accept_documents_from_flagship": {
+    "accepted_mime_types": ["*/*"],
+    "max_number_of_files": 10,
```

Comment:
@Crash-- same here, we don't have any info about what max number of files would be good. We want as many as possible, but I'm unsure what can be handled. I tested with 10 max on my end